recipe/meta.yaml

```diff
@@ -18,24 +18,27 @@ build:
 
 requirements:
   build:
-    - python >=3.6
+    - python >=3.7
     - pip
     - setuptools_scm
```

Comment:
There are missing setuptools and wheel packages in host. We should explicitly add them.
Many python packages use the build tool - setuptools, and where it is mentioned in pytest-mock see https://github.com/pytest-dev/pytest-mock/blob/3c6b070c319e7a524d96e280af9d4e1f404f2722/setup.py#L4

Reply:
Added wheels and setuptools
https://github.com/AnacondaRecipes/pytest-mock-feedstock/blob/76bffa9f9e3f7fb6785e85da32deb1d80955827d/recipe/meta.yaml#L24-L25